### ISSUE TITLE
max-nesting-depth rule not working

### ISSUE THREAD

**Issue by Andrew-Dyachenko** [stylelint/stylelint#3851]
Hello, I'm just trying to skip error of `max-nesting-depth` rule by extending that rule at **.stylelint.yml**

The error I've got:
`Expected nesting depth to be no more than 1   max-nesting-depth`

How to FIX it? Could you help please?

**Terminal screenshot:**
![img-2018-12-15-11-06-33](https://user-images.githubusercontent.com/10127643/50040692-8a117500-0059-11e9-9ddf-57f4ca2ec670.png)

To extending that, I added next rules:
```yaml
rules:
  max-nesting-depth:
    - 5
    - ignoreAtRules:
      - media
      - supports
      - include
```
***.stylelint.yml* piece screenshot:**
![img-2018-12-15-11-13-16](https://user-images.githubusercontent.com/10127643/50040741-7c102400-005a-11e9-8b84-2f7f23b81fba.png)

but it's not working - not skiping error above

***.stylelint.yml* file:**
```yarn
env:
  browser: true
  node: true
  commonjs: true
  es6: true
  jest: true
extends:
  - eslint:recommended
  - plugin:react/recommended
parserOptions:
  ecmaFeatures:
    jsx: true
  ecmaVersion: 2018
  sourceType: module
plugins:
  - react
settings:
  react:
    pragma: React
    version: '16.0'
rules:
  max-nesting-depth:
    - 5
    - ignoreAtRules:
      - media
      - supports
      - include
  indent:
    - error
    - tab
  linebreak-style:
    - error
    - unix
  quotes:
    - error
    - single
    - allowTemplateLiterals: true
  semi:
    - error
    - never
```

_Project bootstraped with Create React App_
